test/Autocomplete.spec.js

```diff
@@ -36,24 +37,38 @@ describe('<Autocomplete />', () => {
 
   describe('initialises', () => {
     const autocompleteInitMock = jest.fn();
+    const autocompleteUpdateDataMock = jest.fn();
     const autocompleteInstanceDestroyMock = jest.fn();
     const autocompleteMock = {
       init: (el, options) => {
         autocompleteInitMock(options);
         return {
-          destroy: autocompleteInstanceDestroyMock
+          destroy: autocompleteInstanceDestroyMock,
+          updateData: autocompleteUpdateDataMock
         };
       }
     };
     const restore = mocker('Autocomplete', autocompleteMock);
+
     beforeEach(() => {
       autocompleteInitMock.mockClear();
       autocompleteInstanceDestroyMock.mockClear();
     });
 
+    afterAll(() => {
+      restore();
```

Comment:
let's move this in the `beforeEach` to make sure that the mocks are cleared for each test case

Reply:
Actually we clear the mocks in `beforeEach`. But I forgot to clear the `autocompleteUpdateDataMock`. Just added that.
Moreover I've added `expect(autocompleteUpdateDataMock).not.toHaveBeenCalled();` to the `calls Autocomplete` `test`.

Reply:
so do we need to keep the `afterAll` `restore`?

Reply:
Yep, have a look yourself at others `spec`. We always `restore()` `afterAll`

Reply:
Cool, thanks for checking 👍

Reply:
;)